src/components/dropdown-menu/DropdownMenu.js

```diff
@@ -125,7 +127,10 @@ class DropdownMenu extends React.Component<Props, State> {
                 break;
 
             case 'Escape':
```

Comment:
Why didn't we use `KEYS.escape` etc. from `constants.js` here? Can you correct them?

Reply:
Yep!  I'll update them, good idea.